### PR TITLE
4.3 field support

### PR DESCRIPTION
**Pull request by ctzsnooze**
Provides full support for all the new field names in 4.3 in the Log Header information screen.

Includes the changes from #535 and #521, which I will close now, as well as supporting the changes made in betaflight PR's [10978 lowpass field names](https://github.com/betaflight/betaflight/pull/10978) and [10856 rpm filter names](https://github.com/betaflight/betaflight/pull/10856) and others.

Fixes an issue with incorrect RPM readings in the Dynamic Idle debug, which were 4 times lower than they should have been.

Tested and ready to merge.

Very useful for anyone working with 4.3 logs after all the changes we've made.

Extensive re-arrangement of fields and CSS changes as well.  The most useful tuning fields are at the top. Fields are entered, and the scrolling widgets are removed.

Screenshot:

Screen Shot 2021-10-31 at 18 12 57

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Updated to be fully compatible with [#11061 ](https://github.com/betaflight/betaflight/pull/11061)

Checked and ready to merge.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
